flink-table-store-core/src/main/java/org/apache/flink/table/store/file/FileStoreOptions.java

```diff
@@ -155,6 +155,14 @@ public class FileStoreOptions implements Serializable {
                             "Open file cost of a source file. It is used to avoid reading"
                                     + " too many files with a source split, which can be very slow.");
 
+    public static final ConfigOption<String> BUCKET_KEY =
+            ConfigOptions.key("bucket-key")
+                    .stringType()
```

Comment:
Add a description of how to specify the bucket key? Users do not know whether the delimiter is `,` or `;` unless they refer to the source code.